### ISSUE TITLE
Forum: emoji size out of proportion in default theme

### ISSUE THREAD

**Issue by luzpaz** [FreeCAD/FreeCAD-Homepage#61]
Issue: https://forum.freecadweb.org/viewtopic.php?t=32722
![image](https://user-images.githubusercontent.com/4140247/102761776-5868ce00-4345-11eb-9741-6d9063773c56.png)

Console warning:
```
This page uses the non standard property “zoom”. Consider using calc() in the relevant property values, or using “transform” along with “transform-origin: 0 0”. viewtopic.php
Source map error: Error: request failed with status 404
Resource URL: https://forum.freecadweb.org/styles/freecad/theme/stylesheet.css?assets_version=30
Source Map URL: stylesheet.css.map
```

This is the expected behavior: switch over to the elegance theme (i'm using the Firefox 'Dark Reader' addon hence the dark theme)
![image](https://user-images.githubusercontent.com/4140247/102761894-8cdc8a00-4345-11eb-9752-8dbffe282f64.png)

**Comment by luzpaz** [comment]
@yorikvanhavre the class selectors are `.emoji .smilies`

**Comment by luzpaz** [comment]
This is still an issue, reopening

**Comment by luzpaz** [comment]
Maybe the `_common.scss` files are being overwritten ?

**Comment by luzpaz** [comment]
Ok, it just needed to be pushed to the server. This issue is fixed!

**Comment by luzpaz** [comment]
stylesheet doesn't apply to those not logged in.

1. log out of forum
2. browse https://forum.freecadweb.org/viewtopic.php?f=3&t=56319#p484442

**Comment by luzpaz** [comment]
@AR795 sorry to ping you. Could you weigh in on this?

**Comment by luzpaz** [comment]
![image](https://user-images.githubusercontent.com/4140247/113228568-254aaa80-9263-11eb-9110-75c70ecde7b3.png)

**Comment by ipatch** [comment]
logged in and emojis are still rendering larger than normal.

![image](https://user-images.githubusercontent.com/613805/114604452-82c8f900-9c5e-11eb-808d-e989a260b1f0.png)

**Comment by luzpaz** [comment]
seriously?

**Comment by ipatch** [comment]
> seriously?

as a heart attack. 💔

**Comment by ipatch** [comment]
it's not related to failing to load the sourcempa is it.

![image](https://user-images.githubusercontent.com/613805/114607980-caea1a80-9c62-11eb-8378-7dedf3abf7a9.png)

**Comment by luzpaz** [comment]
Can you make sure by logging in and hard refreshing pages that have the 'newer' emojis on them?
Also would you mind experimenting with different forum themes (changing them in the settings)?

**Comment by ipatch** [comment]
just tried with vivaldi, chrome, and firefox, logged in, not logged in still the same. **is there a github repo that hosts the phpbb code for the forum?** i might be able to take a look at the source and see if i can find anything out of the norm.

![image](https://user-images.githubusercontent.com/613805/115595144-32741b80-a29c-11eb-9743-3913bd0d68c7.png)

**Comment by luzpaz** [comment]
@yorikvanhavre can you weigh in?

**Comment by ipatch** [comment]
i forgot i posted this thread in the forum a while back,

https://forum.freecadweb.org/viewtopic.php?f=8&t=52663

so i guess all the styling for the forum and wiki are contained within this git repo. 👍

**Comment by luzpaz** [comment]
Indeed. I have merge permissions on this repo...so feel free to submit PRs and I'll test them.

**Comment by ipatch** [comment]
that emoji class you added to the stylesheet isn't being added to the stylesheet when i visit the freecad forum, as evident when viewing the assets for the forum in this image below,

![image](https://user-images.githubusercontent.com/613805/115775654-c61b1a00-a378-11eb-8179-c11c355259c4.png)

if you do a search for `.inner:after` in the sources tab of the browser tools you'll see the `.emoji` class is never added to the stylesheet.

as evident from viewing the commit you made,

https://github.com/FreeCAD/FreeCAD-Homepage/blob/master/forum/styles/freecad/theme/_common.scss#L1936

the emoji class `.emoji` should appear within the stylesheet but does not. it also appears the styles are using sass ie. `.scss` which i believe need to be precompiled at some point before the can be served up to a normal (chrome,firefox) web browser before they well be rendered.

so if i had to take a shot in the dark maybe the assets haven't been compiled since you pushed your changes to this repo.

i don't know how this whole infrastructure works, in hwo the actual server picks the code from this repo and then renders it, so i can't really comment on that. but i hope this helps in further understanding of what is going on.

**Comment by luzpaz** [comment]
@kkremitzki you have experience with sass?

**Comment by kkremitzki** [comment]
Afraid not. This broke again because there were breaking changes to the theme our FreeCAD and FreeCAD-Dark themes are based on... it would be good to have something a little more maintainable.

**Comment by hyarion** [comment]
This seems to have been fixed now
